src/pages/dashboard/components/ExpiredQuotes.tsx

```diff
@@ -52,36 +48,54 @@ export function ExpiredQuotes() {
     {
       id: 'date',
       label: t('date'),
-      format: (value) => date(value, dateFormat),
+      format: (value) => value && dayjs(value).format('MMM DD'),
     },
     {
       id: 'amount',
       label: t('amount'),
-      format: (value, quote) =>
-        formatMoney(
-          value,
-          quote.client?.country_id || company.settings.country_id,
-          quote.client?.settings.currency_id || company.settings.currency_id
-        ),
+      format: (value, quote) => (
+        <Badge variant="light-blue">
+          {formatMoney(
+            value,
+            quote.client?.country_id || company.settings.country_id,
+            quote.client?.settings.currency_id || company.settings.currency_id
+          )}
+        </Badge>
+      ),
     },
   ];
 
   return (
     <Card
       title={t('expired_quotes')}
-      className="h-96"
-      padding="small"
-      withScrollableBody
+      className="h-96 relative"
       withoutBodyPadding
+      withoutHeaderBorder
     >
-      <DataTable
-        resource="quote"
-        columns={columns}
-        endpoint="/api/v1/quotes?include=client&client_status=expired&without_deleted_clients=true&per_page=50&page=1&sort=id|desc"
-        withoutActions
-        withoutPagination
-        withoutPadding
-      />
+      <div className="pl-6 pr-4">
+        <DataTable
+          resource="quote"
+          columns={columns}
+          className="pr-4"
+          endpoint="/api/v1/quotes?include=client&client_status=expired&without_deleted_clients=true&per_page=50&page=1&sort=id|desc"
+          withoutActions
+          withoutPagination
```

Comment:
Can we bundle this in some object perhaps? Perhaps:

interface StyleOptions {
   withoutFoo: boolean
}

Then pass it <DataTable styleOptions={{ withoutFoo: true }} />

Reply:
Yes, it's a good idea. I've grouped all the new styling properties under the `StyleOptions` object. However, there are some properties, like `withoutPagination`, that are used throughout the entire application. I believe it wouldn't be a good idea to modify them in this pull request because it would require changing the `DataTable` component in other parts of the app as well. Here's how the `StyleOptions` object looks now:

```
interface StyleOptions {
  withoutBottomBorder?: boolean;
  withoutTopBorder?: boolean;
  withoutLeftBorder?: boolean;
  withoutRightBorder?: boolean;
  headerBackgroundColor?: string;
  thChildrenClassName?: string;
  tBodyStyle?: CSSProperties;
  thClassName?: string;
  tdClassName?: string;
  addRowSeparator?: boolean;
}
```